packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -50,6 +63,76 @@ const TextStyle _kLargeTitleTextStyle = TextStyle(
   color: CupertinoColors.black,
 );
 
+// There's a single tag for all instances of navigation bars because they can
+// all transition between each other (per Navigator) via Hero transitions.
+const _HeroTag _heroTag = _HeroTag();
+
+class _HeroTag {
+  const _HeroTag();
+  // This class exists because the Gallery smoke test wants all widgets from
+  // debugDumpApp() to have a toString();
+  @override
+  String toString() => "Cupertino navigation bars' Hero tag";
```

Comment:
I think this might be mildly clearer as "Hero tag for Cupertino navigation bars" or just "Navigation bar hero" or some such. The possessive apostrophe is grammatically perfect but will look confusing in tree dumps.